hw/ip/spi_host/doc/_index.md

```diff
@@ -445,6 +463,60 @@ edge: [],
 }
 {{< /wavejson >}}
 
+{{< wavejson >}}
```

Comment:
No substantial changes to documentation, except to add more wavedrom figures to help with implementation.  

Documentation deltas will come in a follow-on commit